appletree/plugins/reconstruction.py

```diff
@@ -118,12 +118,23 @@ def simulate(self, key, parameters, s1_area, s1_correction):
         return key, cs1
 
 
+@export
+class cS2_wo_elifecorr(Plugin):
```

Comment:
Do we need this plugin? @zihaoxu98  If needed, what can be the name `cS2_wo_elifecorr` is neither camel nor snake case.

Reply:
@Tuzhibaevsky was initially developing this for S2-only events like L-shell's charge yield measurement.

Reply:
Would `cS2WoElifecorr` be OK?

Reply:
@Tuzhibaevsky Yes, should be OK.